data-model/SDP-sample-records/work_sample1.json

```diff
@@ -0,0 +1,34 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/work/10",
+  "label": "Hexaemeron (Basil of Caesarea)",
+  "entity_type": "work",
+  "uniform_name": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron",
+  "description": "A commentary on the six days of Creation. The first of its kind, this style of commentary, called Hexaemeron, flourished in late antiquity, and later commentaries drew on Basil's as a model.",
+  "authority_files": [{
+      "id": "https://haf.vhmml.org/work/320756617003",
+      "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron"
+    },
+    {
+      "id": "https://viaf.org/viaf/181934772",
+      "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. | Hexaemeron"
+    }
+  ],
+  "associated_dates": [{
+    "type": "creation",
+    "date_human_readable": "mid-4th c. CE",
+    "date_normalized": "0325/0375"
+  }],
+  "language": "Greek",
+  "genre": ["Theological Works", "Commentary, Biblical", "Commentary, Old Testament", "Commentary, Genesis"],
+  "associated_entities": [{
+      "type": "person",
+      "relation": "author",
```

Comment:
or 'has-author'? Should this be a dedicated field? do we need more specifics for if the author is certain or not, disputed, etc.? Attributed authors will be a backlink